src/components/chip/test/chip.spec.tsx

```diff
@@ -0,0 +1,23 @@
+import { newSpecPage } from '@stencil/core/testing';
+import { Chip } from '../chip';
+
+describe('bds-chip', () => {
+  let page;
+
+  beforeEach(async () => {
+    page = await newSpecPage({
+      components: [Chip],
+      html: `<bds-chip></bds-chip>`,
+    });
+  });
+
```

Comment:
add tests to props and the event emitter

Reply:
fix